### PR TITLE
[feature/footer] add footer component

### PR DESCRIPTION
**Pull request by bartlomiej-sacha**
Following changes were made:

- footer component created

### CODE REVIEW

File: src/components/Footer/Footer.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+            <li className={'py-1'}> friday: 9:00-22:00</li>
+          </ul>
+        </Col>
+        <Col xs={12} md={3} xl={2} className={'  '}>
+          <h3 className={'font-weight-bold'}>Locations</h3>
+          <ul>
+            <li>
+              <p>
+                Location 1:
+                <br /> 1475 Spinnaker Lane
```
Comment (bartlomiej-sacha) [inline src/components/Footer/Footer.js:27]: Is it okay to put p tags in li tags?

Comment (Lukortech) [reply src/components/Footer/Footer.js:27]: I haven't heard about it being a bad practice but maybe there's someone smart in the web who touched the subject. I don't see a reason why would you not do that if your li element consists of multiple paragraphs. In other cases why even use p?

File: src/sass/_footer.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.footer {
+  background-color: black;
```
Comment (Lukortech) [inline src/sass/_footer.scss:2]: I think you can extend these properties by Bootstrap classes like bg-dark and such :)

File: src/sass/style.scss [hunk 3]
```diff
@@ -1,8 +1,9 @@
...
-@import "slider-item";
+@import 'reset';
+@import 'variables';
+@import 'typography';
+@import 'base';
+@import 'header';
+@import 'carousel';
+@import 'circle-item';
+@import 'slider-item';
+@import 'footer';
```
Comment (Lukortech) [inline src/sass/style.scss:9]: In most IDEs you have an option to sort imports on save. I would highly recommend that :)

**Review by Lukortech — APPROVED** [review]

**Pull request merged**